### ISSUE TITLE
Tooltip for links in embedded PDF viewer flashes a black box

### ISSUE THREAD

**Issue by AnomalRoil** [texstudio-org/texstudio#1113]
### Environment
- TeXstudio: 2.12.22 (git 2.12.22)
- Qt: Qt Version 5.12.1, compiled with Qt 5.12.1 R
- OS: Windows 10
- TeX distribution: Miktex

### Expected behavior
When using the embedded PDF viewer to view the PDF, and hovering a link in the PDF made using the hyperref package, I would expect a tooltip to display the actual URL it points to.
Something like this:
![image](https://user-images.githubusercontent.com/10077203/83003390-89cb2680-a00e-11ea-9ffa-ec7b9ff34985.png)

### Actual behavior 
But on Windows, when hovering over a link in the PDF preview pane, here is what first appears before the actual tooltip:
![image](https://user-images.githubusercontent.com/10077203/83003109-25a86280-a00e-11ea-9bf5-c83cc66ddac3.png)
Notice that interestingly the black box width depends on the actual final tooltip width.
It stays for a second and then it changes for the actual tooltip, which look correct:
![image](https://user-images.githubusercontent.com/10077203/83003390-89cb2680-a00e-11ea-9ffa-ec7b9ff34985.png)

Notice **this is not the case on my Linux machine** running TeXstudio 2.12.22 (but using Texlive), there it just works as expected.

### How to reproduce
```
\documentclass[]{article}
\usepackage{hyperref}

\begin{document}
\url{https://speakerdeck.com/pwntester/attacking-net-serialization}\\

This is a \href{https://speakerdeck.com/pwntester/attacking-net-serialization}{test}.
\end{document}
```

**Comment by sunderme** [comment]
seems to be okay on linux

**Comment by AnomalRoil** [comment]
Yup, it's not the case on my Linux computer. Only impacting Windows (10?), AFAICT, but I didn't test on MacOS.

**Comment by muzimuzhi** [comment]
Works as expect on macOS

**Comment by ZurMaD** [comment]
Try to upload to new versions. Working perfectly on Windows 10 - B2004 with TexStudio 3.0.4.
![image](https://user-images.githubusercontent.com/28235457/96352946-4f315b80-108d-11eb-812a-3a67777c63e2.png)

**Comment by AnomalRoil** [comment]
This still reproduces using 3.0.1 on Windows. I'm not sure where 3.0.4 is available? The latest beta (which I haven't tried) [seems to be 3.0.2](https://github.com/texstudio-org/texstudio/releases), AFAICT.

The black box just flashes: it stays for a second and then it changes for the actual tooltip, which look correct:

**Comment by ZurMaD** [comment]
> This still reproduces using 3.0.1 on Windows. I'm not sure where 3.0.4 is available? The latest beta (which I haven't tried) [seems to be 3.0.2](https://github.com/texstudio-org/texstudio/releases), AFAICT.
> 
> The black box just flashes: it stays for a second and then it changes for the actual tooltip, which look correct:

Sorry for that, was 3.0.1. Another way to check if its something wrong is to do the same thing on Overleaf.com

**Comment by AnomalRoil** [comment]
Overleaf does not show any tooltip when hovering a link in the preview pane.

But notice that TexStudio on Linux does not exhibit this flashing.
